test/test_universal_functions.jl

```diff
@@ -38,7 +38,7 @@ universal_functions(uft, L) = UF.universal_func(uft, L, create_uf_parameters(tom
         FT = Float32
         ζ = (-FT(1), FT(0.5) * eps(FT), 2 * eps(FT))
         for L in (-FT(10), FT(10))
-            for uft in (UF.GryanikType(), UF.GrachevType(), UF.BusingerType())
+            for uft in (UF.GryanikType(), UF.GrachevType(), UF.BusingerType(), UF.ChengType())
```

Comment:
Test fails with : 
  ```Got exception outside of a at-test
  type NamedTuple has no field Pr_0_Cheng```
Can you please verify the parameter assignments @gdecker1?

Reply:
Are you using the latest version of ```CLIMAParameters.jl```? I have to manually add the latest commit on the main branch of ``CLIMAParameters.jl``` with ```add CLIMAParameters#main``` before running tests, otherwise I get the exception above.

Reply:
Ah - we may need to request a ClimaParameters release so that we don't have to track the `main` branch manually.